homework.md

```diff
@@ -44,7 +46,7 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
```

Comment:
JavaScript: it helips you to make your web page more interactive in various way.

please leave feedback on my work also. thanks